aptos/token_bridge/sources/bridge_state.move

```diff
@@ -129,9 +123,8 @@ module token_bridge::bridge_state {
         *table::borrow(origin_info_to_wrapped_assets, native_info)
     }
 
-    public fun origin_info(token_address: TokenHash): OriginInfo acquires State {
-        let wrapped_assets_to_origin_info = &borrow_global<State>(@token_bridge).wrapped_assets_to_origin_info;
-        *table::borrow(wrapped_assets_to_origin_info, token_address)
+    public fun origin_info<CoinType>(): OriginInfo acquires OriginInfo {
```

Comment:
assert!(exists<OriginInfo>(account_address))? throw custom error if it doesn't exist?